tests/jerry/regression-test-issue-122.js

```diff
@@ -15,10 +15,11 @@
 
 try
 {
-  print(this, []);
+  print({toString: function() { throw new TypeError("foo"); }}, []);
```

Comment:
I see. Yes, it would be incorrect to replace with simple print.

The change looks good to me.